### ISSUE TITLE
Rest api

### ISSUE THREAD

**Issue by sweetybasak** [priyanka747/Project#1]
Is the project fully using rest api?

**Comment by priyanka747** [comment]
can you explain what you're looking for exactly?

**Comment by priyanka747** [comment]
> Is the project fully using rest api?

this contains  Rest api which only serve data to Android  application also the adminpanel which dont use apis for every function. however, some of page have api calls using jquery

**Comment by sweetybasak** [comment]
can u tell me which pages contains the api calls using jquery?

**Comment by sweetybasak** [comment]
Can i get the sql file for this project?

**Comment by priyanka747** [comment]
> can u tell me which pages contains the api calls using jquery?

yes, its Apiorder.php file in controllers and api_viewallorders.php in view

**Comment by priyanka747** [comment]
yes sure
*Priyanka Thakker*
Full Stack Web Developer
514.703.0747

www.priyankathakker.com
LinkedIn <https://www.linkedin.com/in/priyanka747/>    |  Github
<https://github.com/priyanka747>
